src/App.jsx

```diff
@@ -1,15 +1,15 @@
 import React from 'react';
+import { Sum } from './components/Sum';
+
 import './App.scss';
 
 const App = () => (
   <>
-    <p>Sum of 2 and 3 is 5</p>
-    <p>Sum of -5 and 5 is 0</p>
-    <p>Sum of 10 and 0 is 10</p>
-    <p>Sum of 0 and 5 is 5</p>
-    <p>Sum of 0 and 0 is 0</p>
-    {/* Replace paragraphs with Sum componets */}
-    {/* And remove comment lines :) */}
+    <Sum a={2} b={3} />
+    <Sum a={5} b={-5} />
+    <Sum a={10} b={0} />
+    <Sum a={0} b={5} />
+    <Sum a={0} b={0} />
```

Comment:
Please reread the task and pay attention to the last three components as they should be made in a different way compared to how you did them.

App should contains the next Sum components:
    a = 2 and b = 3
    a = 5 and b = -5
    just a = 10
    just b = 5
    no params at all